packages/simcore-sdk/tests/conftest.py

```diff
@@ -74,3 +76,11 @@ async def _mocked_function(*args, **kwargs) -> None:
         pass
 
     return _mocked_function
+
+
+@pytest.fixture
+def mock_uuid4(mocker: MockerFixture) -> None:
+    mocker.patch(
+        "simcore_sdk.node_ports_common.data_items_utils.uuid4",
+        return_value=UUID(int=0),
+    )
```

Comment:
you could use a mocker.spy item for this.
This would be less of a finicky thing to follow up, that the return value here is linked to the constant you are using.
https://pytest-mock.readthedocs.io/en/latest/usage.html#spy

Reply:
I don't see how this aids me with enforcing a certain value generated by `uuid4()` outside the scope of a test, at module level.

Unfortunately the tests assume that this value is always constant, since they are inherited from the previous implementation.

The issue here is not the serialisation or the `{UUID}` value of inside the path: `/tmp/simcore/{UUID}/rest-of-file-or-folder`.
The tests require this `{UUID}` to be constant.

To avoid this requirement I have to use a lot of time to redesign a lot of tests, which I would like to avoid.